### PR TITLE
media-libs/libsoundio: version bump to 2.0.0

### PR DESCRIPTION
**Pull request by diogocp**
Closes: https://bugs.gentoo.org/681478

### CODE REVIEW

**Comment by a17r** [comment]
Did you check all revdeps not to break against this new major version?

**Comment by diogocp** [comment]
Upstream claims the API is backwards compatible: https://github.com/andrewrk/libsoundio/issues/208#issuecomment-469035477

Only revdep seems to be media-sound/lmms. Didn't test.

File: media-libs/libsoundio/libsoundio-2.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DENABLE_ALSA=$(usex alsa)
+		-DENABLE_COREAUDIO=$(usex coreaudio)
+		-DENABLE_JACK=$(usex jack)
+		-DENABLE_PULSEAUDIO=$(usex pulseaudio)
+		-DENABLE_WASAPI=no
+		-DBUILD_STATIC_LIBS=$(usex static-libs)
+		-DBUILD_EXAMPLE_PROGRAMS=$(multilib_native_usex examples)
+		-DBUILD_TESTS=no
```
Comment (mgorny) [inline media-libs/libsoundio/libsoundio-2.0.0.ebuild]: Does this mean that the package has no ebuild-usable tests, or that you failed to run them?

Comment (diogocp) [reply media-libs/libsoundio/libsoundio-2.0.0.ebuild]: They were not usable back when I added the ebuild (see https://github.com/gentoo/gentoo/pull/65#issuecomment-137572963), and I don't think there has been any change, although I didn't check.

**Pull request closed without merging**